src/Tests/Microsoft.NET.Build.Tests/GivenThatWeWantToBuildWithGlobalJson.cs

```diff
@@ -16,29 +18,60 @@ public class GivenThatWeWantToBuildWithGlobalJson : SdkTest
         public GivenThatWeWantToBuildWithGlobalJson(ITestOutputHelper log) : base(log)
         {}
 
-        [Fact]
-        public void It_fails_build_on_failed_sdk_resolution()
+        [FullMSBuildOnlyTheory]
+        [InlineData(true)]
+        [InlineData(false)]
+        public void It_fails_build_on_failed_sdk_resolution(bool runningInVS)
         {
-            var fakePath = "fakePath";
-            TestProject testProject = new TestProject()
+            var prevIncludeDefault = Environment.GetEnvironmentVariable("MSBUILDINCLUDEDEFAULTSDKRESOLVER");
+            var prevAdditionalFolder = Environment.GetEnvironmentVariable("MSBUILDADDITIONALSDKRESOLVERSFOLDER");
```

Comment:
What about setting up these environment variables for all tests (when running on full framework)?  That was flaky last time we tried it, but I'd still like to try to turn that back on.  It doesn't necessarily have to be in this PR though.

Reply:
Actually, it looks like we're already setting this up: https://github.com/dotnet/sdk/blob/release/5.0.1xx/src/Tests/Microsoft.NET.TestFramework/ToolsetInfo.cs#L154

So it seems like setting this variable here may be redundant.